templates/web/base/report/inspect/public_update.html

```diff
@@ -7,8 +7,34 @@
               [% loc('Save with a public update') %]
             </label>
           </p>
+          <div id="public_update_form_fields">
+          [% IF c.cobrand.allow_photo_upload %]
```

Comment:
This would seem a good time to factor out this whole IF statement (which also occurs identically in at least three other places) into its own template?